app/models/job_invocation_composer.rb

```diff
@@ -75,6 +75,29 @@ def targeting_params
       api_params.slice(:targeting_type, :bookmark_id, :search_query).merge(:user_id => User.current.id)
     end
 
+    def triggering_params
+      raise ::Foreman::Exception, _('Cannot specify both recurrence and scheduling') if api_params[:recurrence].any? && api_params[:scheduling].any?
+
+      if api_params[:recurrence].any?
+        {
+          :mode          => :recurring,
+          :cronline      => api_params[:recurrence][:cron_line],
```

Comment:
But aligned hash :rocket: are so pretty :heart_eyes: I get what you're saying though, fixed